tools/log-collector/helper.go

```diff
@@ -71,6 +73,9 @@ func aggregateEvents(eventObjects unstructured.UnstructuredList,
 		}
 
 		namespace, _, nErr := unstructured.NestedString(eve.Object, "involvedObject", "namespace")
+		if namespace == "" {
```

Comment:
This `if` block should be placed after checking `nErr`

Reply:
done